### ISSUE TITLE
TypeScript - Property Atom does not exist on type TruncateMarkup

### ISSUE THREAD

**Issue by Ebram-Tharwat** [parsable/react-truncate-markup#64]
When trying to use `TruncateMarkup.Atom` with TypeScript I get the following error message:
`Property 'Atom' does not exist on type 'typeof TruncateMarkup'`.

Looking at `index.d.ts`, I see it is **missing** the export/definition of `Atom` .. Could you please help me to overcome that issue?

**Comment by Ebram-Tharwat** [comment]
I found a _hacky_ solution:
- Declare the `Atom` component as `const TruncateMarkupAtom = (TruncateMarkup as any).Atom;` _(notice casting to `any`)_
- Use it as a normal component 
```javascript
list.map((item, index) => (
  <TruncateMarkupAtom key={index}>
    {item}
    <br />
  </TruncateMarkupAtom>
));
```

**Comment by patrik-piskay** [comment]
Fixed in [v5.1.0](https://github.com/parsable/react-truncate-markup/releases/tag/5.1.0)
